.github/ISSUE_TEMPLATE/ticket_template1.md

```diff
@@ -0,0 +1,27 @@
+🚨Please review the [guidelines for ticket creation](../HOWTOWRITEATICKETS.md) if needed
+
+## Problem / Motivation
+Describe your issue here. You can use the user story format.
+- As a ... Who?
+- I would like ... what?
+- So that ... why?
+
+
+## Proposed solution and tasks
+We could build this.... and it will solve that...
+
+List of tasks for this ticket to be complete:
+- [ ] A descriptive of a task
+- [ ] A description
+
+## Clarification needed
```

Comment:
I would say this would be asked by anyone reading the issue, not writing it. The author should have made it clear enough.

Reply:
Your point is useful and I will add a description to explain why this section is very important for anybody writing a ticket whether you are a developer or a non-technical stakeholder. If someone has a very clear idea about what needs to be built then this person should write their assumptions because they might be wrong or they might help others to understand the requirements. Assumptions could trigger a useful discussion too. So this section acts as a reminder for the writer to clarify any points that might be blurry for him or the readers. Most of the Libero contributors won't have as much knowledge about the product as an eLife developer.